plugins/dfu/dfu-device.c

```diff
@@ -689,6 +710,10 @@ dfu_device_set_quirks (DfuDevice *device)
 		}
 	}
 
+	/* Atmel bootloader */
+	if (vid == 0x03eb)
```

Comment:
It would be good to put this into a `#define` for the vendor instead of comment.

Reply:
I've avoided this in the set_quirks() function as we only need it once, and I'm not sure it adds to the readability.

Reply:
Well maybe not now, but i was thinking if the quirks grows a lot then it would make it more readable later to do it that way.

Reply:
I think this is solved by https://github.com/hughsie/fwupd/pull/293 :)

Reply:
True :)